src/evaluator/types.ts

```diff
@@ -1,12 +1,12 @@
 import { Result } from '../utils';
-import { SExpr, SListStruct } from '../sexpr';
+import { SExprT } from '../sexpr';
```

Comment:
`SExprT` means S-expr term?

Reply:
SExprT means SListStruct, i was going for a name like "generic S expression"

any ideas?